src/audio/buffer.c

```diff
@@ -40,8 +40,8 @@ struct comp_buffer *buffer_new(struct sof_ipc_buffer *desc)
 				   "could not alloc structure");
 		return NULL;
```

Comment:
@dabekjakub please fix your commit message:

1) Please use imperative mood. Use "memory: Add support for getting aligned buffer heap memory" (or something similar)
2) Please explain why some buffers need to be aligned.

You can follow the guide here on how to write a good commit message https://chris.beams.io/posts/git-commit/#seven-rules

Reply:
Changed commit message and will split commits